src/UI/pages/Movie/index.tsx

```diff
@@ -1,5 +1,34 @@
 import React, { FC } from "react";
 
-const Movie: FC = () => <p className="text-white">Movie</p>;
+import { useParams } from "react-router-dom";
+
+import { useGetMovieQuery } from "../../../store/api";
+
+const Movie: FC = () => {
+  const params = useParams();
+  console.log(params.id);
+  const id = params.id?.replace(":id", "");
+
+  const { data: movie, error, isLoading } = useGetMovieQuery(id);
+
+  return (
+    <div>
+      {error && <p className="text-red-500">Error!</p>}
+      {isLoading && <p className="text-red-500">Loading...</p>}
+
+      {movie && (
+        <>
+          <h1 className="text-white">{movie.title}</h1>
+          <p className="text-white">{movie.release_dare}</p>
```

Comment:
create function ( like we did in Starwars ) so that it would change release_dare to releaseDare